### PR TITLE
Adjust L2 tx wait time

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-1984 (continuation)

## What changed (plus any additional context for devs)

- changed the gas data internal formatting for L2s to follow `normal`, `fast` and `urgent` speeds
```
interface GasPricesAPIData {
  normal: Numberish;
  normalWait: Numberish;
  fast: Numberish;
  fastWait: Numberish;
  urgent: Numberish;
  urgentWait: Numberish;
}
```
I didn't want to modify this interface for optimism and arbitrum (these only have `normal` speed), since we may want to add these options in the future, instead I just passed the same `normal` speed data for `fast` and `urgent`.

- removed `parseGasPricesPolygonGasStation` specific method to parse polygon data to simplify how we parse L2 gas prices so now it's not depending on network https://github.com/rainbow-me/rainbow/compare/@esteban/adjust-l2-tx-wait-time?expand=1#diff-d6624fa29be7d9dfa3819f22204ec01a0e468f7537771f4515740d6f3194be67L206

- set 2 blocks as number of confirmation times for `normal` speed in L2 txs (thumb rule and hardcoded, we can change it), from that the time of confirmation for 
    polygon: 2.5/3 secs per block - 6 secs normal speed, 2 blocks, and 3 secs for fast and normal
    arbitrum: 4 secs per block - 8 secs normal speed, we don't have other speed here
    optimism: 10/15 secs per block - 20 secs normal speed, we don't have other speed here

## PoW (screenshots / screen recordings)

http://recordit.co/8pN9agDeeE
 
## Dev checklist for QA: what to test

check that the time for assets in different networks work as expected

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?

### CODE REVIEW

File: src/redux/gas.ts [hunk 1]
```diff
@@ -350,15 +362,16 @@ export const gasPricesStartPolling = (network = Network.mainnet) => async (
...
-      fastWait: 0.2,
-      safeLow: gasPriceGwei,
-      safeLowWait: 1,
+      fast: normalGasPrice,
+      fastWait: 0.34,
+      normal: normalGasPrice,
+      // 20 secs
+      normalWait: 0.34,
+      urgent: normalGasPrice,
+      urgentWait: 34,
```
Comment (jinchung) [inline src/redux/gas.ts:374]: is this supposed to be 0.34 like the other `normalWait` and `fastWait` ?

Comment (estebanmino) [reply src/redux/gas.ts:374]: oh yeah, we're not using those values but I'll change it 👌

**Review by jinchung — APPROVED** [review]
🌮 

Checked it out on sim, looks good!

**Review by skylarbarrera — APPROVED** [review]
👌

**Pull request merged**